dokka-runners/dokka-gradle-plugin/src/test/kotlin/DokkaPluginTest.kt

```diff
@@ -115,5 +120,74 @@ class DokkaPluginTest : FunSpec({
                 }
             }
         }
+
+        context("DokkaPublications") {
```

Comment:
These tests are missing `test` blocks and `./gradlew :dokka-gradle-plugin:test --tests 'org.jetbrains.dokka.gradle.DokkaPluginTest' --info --rerun` doesn't put these in the junit XML output. Is this correct?

Reply:
Thanks! Fixed. `checkAll() {]` must be inside a `test() {}` block.